ui/main/src/app/services/export.service.ts

```diff
@@ -9,18 +9,21 @@
  */
 import * as XLSX from 'xlsx';
 import {saveAs} from 'file-saver-es';
+import {AOA2SheetOpts, JSON2SheetOpts} from 'xlsx';
```

Comment:
Sonar is complaning about a duplicate import: https://sonarcloud.io/project/issues?id=org.lfenergy.operatorfabric%3Aoperatorfabric-core&open=AXqMB3eqWV0JQmXFQJNh&pullRequest=1432&resolved=false&types=CODE_SMELL

Reply:
ok done